CK2Plus_expanded/events/000_CK2Plus_startup_events.txt.txt

```diff
@@ -0,0 +1,62 @@
+
+namespace = PlusStartup
+
+#######################################
+#
+# CK2Plus Startup Events
+#
+#######################################
+#
+# CATALOG
+#
+# 001 - 100: General events
+# 101 - 200: Toby Determined
+#
+#######################################
+
+#######################################
+#
+# 001 - 100: General events
+#
+#######################################
+#
+# INDEX
+#
+# 001: Assign local culture to barons
+#
+#######################################
+
+# Assign local culture to barons
+character_event = {
+	id = PlusStartup.001
+	
+	is_triggered_only = yes	# from on_chronicle_start, fires only for the player
+	
+	hide_window = yes
+	
+	immediate = {
+		any_character = { # NOTE WARNING this is really expensive, but only used once on startup here, so it is acceptable. Don't use it elsewhere without good reason.
+			limit = {
+				# NOTE for the future: If/once we add playable barons, we need to adjust this event to not culture flip them, and stay as shown in the lobby.
+				# For this, "has_landed_title = c_baron_t" can check for playable barons according to MightNight.
+				tier == baron
+				is_patrician = no	# We don't want to affect republics, these are done in history
+			}
+			capital_scope = { # The baron's capital province (the province of the barony)
+				reverse_culture = PREV	# Copies the province's culture onto the baron
+			}
+			any_close_relative = {	# Close family should be converted, too
```

Comment:
Can we just do this for anyone in that baron's court?

Reply:
`any_realm_character` would do the trick. It sounds like it includes the top liege, but it's actually for the current character's demesne (according to the wiki)

Reply:
I actually did that in the initial commit.
But after some testing it turned out that barons do not have _any_ courtiers generated at gamestart! They don't even have a family, unless it is defined in history. While I left the family part in for these few cases, converting courtiers is just pointless as there are none at that point.